src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -27,7 +27,8 @@ library BeaconChainProofs {
     uint256 internal constant VALIDATOR_FIELD_TREE_HEIGHT = 3;
 
     uint256 internal constant NUM_EXECUTION_PAYLOAD_HEADER_FIELDS = 15;
```

Comment:
this changes to 17?

Reply:
ah, you are correct!

Reply:
this constant isn't used anywhere though 🤷